### PR TITLE
2020Q4: Add a report on GPL-in-Base

### PR DESCRIPTION
**Pull request by kevans91**
CC @bapt @emaste 

I didn't add bapt to the overall contact, but I can...

### CODE REVIEW

File: 2020q4/basegpl.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## GPL in Base ##
+
+Link:	 [GPL Software in the Base System](https://wiki.freebsd.org/GPLinBase)  
+
+Contact: Ed Maste <emaste@FreeBSD.org>  
+Contact: Kyle Evans <kevans@FreeBSD.org>  
+
+A long-standing goal of the FreeBSD project is for the base system to migrate
+to modern, copyfree or more permissively licensed components.  In this quarter,
+the following components have been successfully removed or replace:
```
Comment (emaste) [inline 2020q4/basegpl.md:10]: replaced

File: 2020q4/basegpl.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  * gdb ([removed](https://cgit.freebsd.org/src/commit/?id=1c0ea326aa6d) in favor of devel/gdb in ports)
```
Comment (emaste) [inline 2020q4/basegpl.md:12]: should mention lldb in the base system too

**Comment by bapt** [comment]
> CC @bapt @emaste
> 
> I didn't add bapt to the overall contact, but I can...

Yes you can given the number of GPL replacement I have been working on ;)

**Comment by debdrup** [comment]
Lemme know when this is ready to merge :)

**Comment by kevans91** [comment]
> Lemme know when this is ready to merge :)

Yup, sorry, I'll add bapt to the contact list in a bit

**Comment by debdrup** [comment]
If that's all that's missing, I can handle that ;)

**Comment by emaste** [comment]
Update looks good to me

**Pull request merged**